### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by xeoneux**
TODO:

- [ ] Update CHANGELOG.md with your change (include reference to issue & this PR)
- [ ] Make sure all of the significant new logic is covered by tests
- [ ] Rebase your changes on master so that they can be merged easily
- [ ] Make sure all tests and linter rules pass

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,9 +64,10 @@ app.use('/graphql', bodyParser.json(), apolloConnect({ schema: myGraphQLSchema }
 app.listen(PORT);
 ```
 
-### HAPI
+### hapi
 ```js
 import hapi from 'hapi';
+import { apolloHAPI } from 'apollo-server';
```
Comment (helfer) [inline README.md]: I believe this should be `ApolloHapi` (note the uppercase)

**Comment by helfer** [comment]
Thanks! Can you fix the typo so I can merge it?

**Comment by xeoneux** [comment]
Ideally yes, it should be `ApolloHapi` but is exported as `ApolloHAPI` in the code.
Ref: https://github.com/apollostack/apollo-server/blob/master/src/integrations/hapiApollo.ts#L17

**Comment by helfer** [comment]
Yeah, agree. For now, let's keep it this way to avoid breaking changes.

Thanks!

**Pull request merged**